lib/Elastica/Query/Filtered.php

```diff
@@ -21,6 +23,8 @@ class Filtered extends AbstractQuery
      */
     public function __construct(AbstractQuery $query = null, AbstractFilter $filter = null)
     {
+        trigger_error('Use BoolQuery instead. Filtered query is deprecated since ES 2.0.0-beta1 and this class will be removed in further Elastica releases.', E_USER_DEPRECATED);
```

Comment:
1. Better to have trigger error even if developer just include file. See for example https://github.com/ruflin/Elastica/blob/master/lib/Elastica/Query/Bool.php